2017/AUG/AUG17/RAINBOWA/RAINBOWA.java

```diff
@@ -0,0 +1,55 @@
+//Made by rachanasoma
+import java.util.*;
+import java.lang.*;
```

Comment:
Please optimize imports, i.e. import only those libraries which are required.
This saves a lot of compilation time.